desktop/cmp/input/Select.js

```diff
@@ -613,7 +634,7 @@ const cmp = hoistCmp.factory(
                 onFocus: model.onFocus,
                 filterOption: model.filterOption,
 
-                ref: model.reactSelectRef
+                ref: composeRefs(model.inputRef, props.inputRef, model.reactSelectRef)
```

Comment:
why does the model need two identical refs? why are we introducing a new prop ref here?  this seems to me to be cruft.

Reply:
Fixed.